### PR TITLE
V2 add redactor docs

### PR DESCRIPTION
**Pull request by shiranr**
Add docs for image redactor

### CODE REVIEW

File: docs/image-redactor/index.md [hunk 1]
```diff
@@ -1,22 +1,113 @@
 # Presidio Image Redactor
 
+***Please notice, this package is still in alpha and not production ready.***
```
Comment (navalev) [inline docs/image-redactor/index.md:3]:
```suggestion
***Please notice, this package is still in beta and not production ready.***
```

**Review by navalev — APPROVED** [review]

File: docs/image-redactor/index.md [hunk 2]
```diff
@@ -1,22 +1,113 @@
...
+
 ## Description
 
-## Table of contents
+The Presidio Image Redactor is a Python based module for detecting and redacting PII
+text entities in images.
+![img.png](img.png)
```
Comment (SharonHart) [inline docs/image-redactor/index.md:9]: already in assets/

File: presidio-image-redactor/README.MD [hunk 3]
```diff
@@ -100,7 +100,7 @@ curl example:
 ```
```
Comment (SharonHart) [inline presidio-image-redactor/README.MD:100]: I would move some of the stuff to docs/image-redactor/index.md , keep this file lean like in the other services

File: docs/image-redactor/index.md (hunk 2 above)
Comment (shiranr) [reply docs/image-redactor/index.md]: Done.

File: presidio-image-redactor/README.MD (hunk 3 above)
Comment (shiranr) [reply presidio-image-redactor/README.MD:100]: I think this one is already very lean. 
I will add a Home as the other have.

File: docs/image-redactor/index.md [hunk 4]
```diff
@@ -1,22 +1,113 @@
...
+
+  The Image-Redactor was tested with a minimal version of 4.0.0
 
-- Running in Python.
-- Running using Docker.
-- Customizing.
-- API reference.
+=== "Using pip"
+    
+    !!! note "Note"
```
Comment (balteravishay) [inline docs/image-redactor/index.md:22]: note "note" ?

File: docs/image-redactor/index.md [hunk 5]
```diff
@@ -1,22 +1,113 @@
...
+    To get started with Presidio-image-redactor,
+    download the package and the `en_core_web_lg` spaCy model:
+    
+    ```sh
+    pip install presidio-image-redactor
+    python -m spacy download en_core_web_lg
+    ```
 
-### Running in Python
+=== "Using Docker"
```
Comment (balteravishay) [inline docs/image-redactor/index.md:33]: nip: consider using standard headers

File: mkdocs.yml [hunk 6]
```diff
@@ -22,7 +22,8 @@ nav:
       - Presidio Anonymizer:
         - Home: anonymizer/index.md
         - Developing PII anonymizers: anonymizer/adding_anonymizers.md
-    - Image anonymization: image-redactor/index.md
+    - Image redaction:
```
Comment (SharonHart) [inline mkdocs.yml:25]:
```suggestion
    - Image Redactor:
```

File: docs/image-redactor/index.md [hunk 7]
```diff
@@ -1,22 +1,113 @@
...
+    cd presidio-image-redactor
+    python app.py
+    # use ocr_test.png as the image to redact, and 255 as the color fill. 
+    # out.png is the new redacted image received from the server.
+    curl -XPOST "http://localhost:3000/redact" -H "content-type: multipart/form-data" -F "image=@ocr_test.png" -F "data=\"{'color_fill':'255'}\"" > out.png
+    ```
+Python script example can be found under:
+/presidio/e2e-tests/tests/test_image_redactor.py
 
 ### API reference
```
Comment (balteravishay) [inline docs/image-redactor/index.md:109]: I think this header indentation is wrong. its currently under ##Getting started.

**Review by balteravishay — COMMENTED** [review]
few header/indent comments

File: docs/image-redactor/index.md [hunk 8]
```diff
@@ -1,22 +1,113 @@
...
+    
+    !!! note "Note"
+        Consider installing the Presidio python packages on a virtual environment like venv or conda.
+    
+    To get started with Presidio-image-redactor,
+    download the package and the `en_core_web_lg` spaCy model:
+    
+    ```sh
+    pip install presidio-image-redactor
+    python -m spacy download en_core_web_lg
```
Comment (SharonHart) [inline docs/image-redactor/index.md:30]: I remember that I saw Omri commenting once that this is not needed when pip installing image-redactor?

**Review by itye-msft — APPROVED** [review]
LGTM

File: docs/image-redactor/index.md (hunk 4 above)
Comment (SharonHart) [reply docs/image-redactor/index.md:22]: This is the syntax for a mkdocs extention, admonition, better formatting notes:
image
Downside is a 'dirtier' markdown

File: docs/image-redactor/index.md (hunk 5 above)
Comment (shiranr) [reply docs/image-redactor/index.md:33]: WDYM by standard header? 
This is part of a menu.

File: docs/image-redactor/index.md (hunk 8 above)
Comment (shiranr) [reply docs/image-redactor/index.md:30]: Not that I am aware of... 
If so, I missed that comment... 
@omri374 should we remove this here?

File: docs/image-redactor/index.md (hunk 7 above)
Comment (shiranr) [reply docs/image-redactor/index.md:109]: Screen Shot 2021-03-01 at 12 23 50

Comment (shiranr) [reply docs/image-redactor/index.md:109]: That's how it currently looks (it is aligned with anonymizer and analyzer).
How should I change it? :)

File: docs/image-redactor/index.md (hunk 5 above)
Comment (shiranr) [reply docs/image-redactor/index.md:33]: Screen Shot 2021-03-01 at 12 27 07

**Review by SharonHart — APPROVED** [review]

File: docs/image-redactor/index.md (hunk 8 above)
Comment (shiranr) [reply docs/image-redactor/index.md:30]: Discussed. 
Thanks :)

File: docs/image-redactor/index.md (hunk 5 above)
Comment (balteravishay) [reply docs/image-redactor/index.md:33]: my bad

File: docs/image-redactor/index.md (hunk 7 above)
Comment (shiranr) [reply docs/image-redactor/index.md]: Done.

**Review by balteravishay — APPROVED** [review]

**Pull request merged**